resources/assets/enderio/lang/de_DE.lang

```diff
@@ -559,58 +1022,106 @@ enderio.gui.conduit.item.roundRobinEnabled=Round-Robin Aktiviert
 enderio.gui.conduit.item.roundRobinDisabled=Round-Robin Deaktiviert
 enderio.gui.conduit.item.priority=Priorität
 
+enderio.gui.conduit.item.fuzzy.disabled=Ungenauer Modus deaktiviert
+enderio.gui.conduit.item.fuzzy.fuzzy_25=Ungenauer Abgleich für 25% Schaden
+enderio.gui.conduit.item.fuzzy.fuzzy_50=Ungenauer Abgleich für 50% Schaden
+enderio.gui.conduit.item.fuzzy.fuzzy_75=Ungenauer Abgleich für 75% Schaden
+enderio.gui.conduit.item.fuzzy.fuzzy_99=Ungenauer Abgleich für alle Schadenswerte
+
 enderio.gui.conduit.item.filterupgrade=Item-Filter einsetzen
 enderio.gui.conduit.item.speedupgrade=Leitungsgeschwindigkeits-Upgrade oder -Downgrade einsetzen
 enderio.gui.conduit.item.speedupgrade2=Maximum der Upgrades ist 15; Maximum der Downgrades ist 1
+enderio.gui.conduit.item.functionupgrade.line1=Mögliche Funktions-Upgrades:
 
 enderio.gui.conduit.redstone.signalColor=Signalfarbe
 enderio.gui.conduit.redstone.color=Farbe
+enderio.gui.conduit.redstone.signalStrengh=Starkes Signal
+enderio.gui.conduit.redstone.signalStrengh.tooltip=Gebe ein starkes Redstonesignal ab 
 
 enderio.gui.conduit.fluid.autoExtract=Auto-Extrahieren
 enderio.gui.conduit.fluid.filter=Filter
+enderio.gui.conduit.fluid.whitelist=Whitelist
+enderio.gui.conduit.fluid.blacklist=Blacklist
 
+enderio.gui.conduit.gas.autoExtract=Auto-Extraktion
 
-//Configuration
-enderio.config.title=Ender IO Configuration
+//Weather
+enderio.gui.weather.task.clear=Klares Wetter
+enderio.gui.weather.task.rain=Regnerisches Wetter
+enderio.gui.weather.task.storm=Gewitter
+enderio.gui.weather.requireditem=Benötigtes Item: %s
 
-enderio.config.power=Power Settings
-enderio.config.power.tooltip=So many numbers to change...
+//Enchants
+enchantment.enderio.soulBound=Seelengebunden
+description.enchantment.enderio.soulBound=Verhindert das Verlieren des Item bei Tod
```

Comment:
"beim Sterben" (Tod is the state of being dead, Sterben is the act of getting there)